src/main/java/io/ebean/config/properties/LoadContext.java

```diff
@@ -44,54 +44,50 @@ InputStream resource(String resourcePath, Loader.Source source) {
     } else {
       File file = new File(resourcePath);
       if (file.exists()) {
-        try {
-          is = new FileInputStream(file);
-          loadedResources.add("file:"+resourcePath);
-        } catch (FileNotFoundException e) {
-          throw new IllegalStateException(e);
```

Comment:
OLD: FileNotFoundException was catched and wrapped in an IllegalStateException
NEW: Method throws IOException